clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -347,34 +280,38 @@ opcode_mix_t::get_category_names(uint category)
 bool
 opcode_mix_t::print_results()
 {
-    shard_data_t total(0);
+    shard_data_t aggregated;
+    shard_data_t *total = &aggregated;
     if (shard_map_.empty()) {
-        total = serial_shard_;
+        // No default copy constructor for shard_data_t because of the
+        // std::unique_ptr, so we resort to keeping a pointer to it.
```

Comment:
Will likely need a similar internal change: please test and prepare ahead of time.

Reply:
Done.